### ISSUE TITLE
Start unnamed pools

### ISSUE THREAD

**Issue by lpil** [SpaceTime-IoT/pgo#79]
Hello!

Would it be possible to remove the requirement for pools to be named?

Currently if we want to dynamically connect to a database (for example, using different databases for different users in a multitenant system) we need to dynamically generate an atom and have some mechanism for recycling these atoms to avoid hitting the atom limit. This can be challenging to implement.

If we could reference the pool using a pid or a reference rather than a name we wouldn't have this issue.

Thanks,
Louis

**Comment by tsloughter** [comment]
Good point. I can't think of anything that would make this not possible. Only "difficulty" may be the API. I'll take a closer look and see what I can do.

**Comment by lpil** [comment]
Thank you!
